fendermint/app/src/metrics/tracing.rs

```diff
@@ -108,6 +127,16 @@ impl<S: Subscriber> Layer<S> for MetricsLayer<S> {
             ParentFinalityCommitted {
                 block_height              => set_gauge   ! &am::TOPDOWN_FINALIZED_BLOCK_HEIGHT,
             },
+            ParentFinalityVoteAdded {
+                block_height              => max_gauge    ! &am::TOPDOWN_FINALITY_VOTE_BLOCK_HEIGHT,
```

Comment:
What if this height goes backwards? I think it would be nice to see that in Prometheus / Grafana. Suggest to lift the `max` filtering, unless there's a reason that I miss.

Reply:
The reason is that votes arrive unordered through gossip. I can receive a vote from Alice about block 123, then a block from Bob about block 99. There's no point jumping up and down as they come randomly, that would make this rather meaningless IMO. You can pick those raw values up from the log if you want to see them. This gauge is merely an indication of what is the highest block we see votes on, so it should, in general, go up, indicating that the group is making progress through the gossiping of votes.

Reply:
Okay, I added both versions with comments. I know you will say it doesn't make sense to keep the max version because you can produce it in Grafana, but in the past I found it convenient to just use `curl` and see what I'm interested in. Grafana will naturally only take a sample of the observations, while a metric for the highest vote received is durable and won't be masked by a followup vote on a lower value. 

In the end I agree that the `set_gauge` version is useful because at the moment if a single Byzantine validator casts a vote on some stupid high value, it can lock the `max` metric for good. Now that will be easy to see and will indicate foul play. On Grafana it would have been down to luck to query the node at the time where the `set` version shows a high value.